packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -1489,7 +1489,7 @@ See: https://github.com/flutter/flutter/issues/30701.
   /// This setter can be used to provide a custom [WindowingOwner].
   ///
   /// {@macro flutter.widgets.binding.window.experimental}
-  @internal
+  @protected
```

Comment:
I'd prefer to keep `@internal` here if possible - it is a strong signal that users should not use this API, even if they're extending widgets binding.

I'm guessing you changed this so that it could be set in `flutter_test` without causing a diagnostic? Could we instead suppress the diagnostic where needed? I see you already added `// ignore_for_file: invalid_use_of_internal_member` - was that not enough?